doc/classes/Geometry3D.xml

```diff
@@ -142,5 +142,12 @@
 				Tests if the segment ([param from], [param to]) intersects the triangle [param a], [param b], [param c]. If yes, returns the point of intersection as [Vector3]. If no intersection takes place, returns [code]null[/code].
 			</description>
 		</method>
+		<method name="tetrahedralize_delaunay">
+			<return type="PackedInt32Array" />
+			<param index="0" name="points" type="PackedVector3Array" />
+			<description>
+				Tetrahedronizes the volume specified by a discrete set of [param points] in 3D space, ensuring that no point lies within the circumsphere of any resulting tetrahedron. The method returns a [PackedInt32Array] where each tetrahedron consists of four consecutive point indices into the [param points] array (resulting in an array with [code]n * 4[/code] elements, where [code]n[/code] is the number of tetrahedra found). If the tetrahedronization is unsuccessful, an empty [PackedInt32Array] is returned.
```

Comment:
So which one is the verb for this? "tetrahedralize" or "tretrahedronize"?

Reply:
Good catch, English isn't my first language and `tetrahedralize` seems to be a neologism of sorts, there is this [discussion](https://linguistics.stackexchange.com/q/29682) on linguistics stack exchange that doesn't seem to have landed on any conclusion; I say we change `tetrahedronization` (which I wrote, my bad) and stick to `tetrahedralization` everywhere else.

I'll submit a commit fixing this soon.